Data/ByteString/Streaming/Internal.hs

```diff
@@ -427,13 +432,12 @@ foldlChunksM f z bs = z >>= \a -> go a bs
       Go m       -> m >>= go a
 {-# INLINABLE foldlChunksM #-}
 
-
-
 -- | Consume the chunks of an effectful ByteString with a natural right monadic fold.
 foldrChunksM :: Monad m => (B.ByteString -> m a -> m a) -> m a -> ByteString m r -> m a
 foldrChunksM step nil bs = dematerialize bs (const nil) step join
 {-# INLINE foldrChunksM #-}
 
+-- | Internal utility for @unfoldr@.
```

Comment:
```suggestion
-- | Internal utility for 'unfoldr'.
```

Reply:
This was to prevent a docs warning. The `unfoldr` in question here isn't in scope.

Reply:
That's fine then, use `@` when necessary, but not backticks, which are markdown, not haddock.